src/gui/editors/PianoRoll.cpp

```diff
@@ -3466,33 +3468,29 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 
 
 // responsible for moving/resizing scrollbars after window-resizing
-void PianoRoll::resizeEvent(QResizeEvent * re)
+void PianoRoll::resizeEvent(QResizeEvent* re)
 {
-	m_leftRightScroll->setGeometry( WHITE_KEY_WIDTH,
-								      height() -
-								SCROLLBAR_SIZE,
-					width()-WHITE_KEY_WIDTH,
-							SCROLLBAR_SIZE );
-	m_topBottomScroll->setGeometry( width() - SCROLLBAR_SIZE, PR_TOP_MARGIN,
-						SCROLLBAR_SIZE,
-						height() - PR_TOP_MARGIN -
-						SCROLLBAR_SIZE );
+	m_leftRightScroll->setGeometry(WHITE_KEY_WIDTH,
```

Comment:
Tabs here are a little off, maybe reduce to 2 tabs instead.

Reply:
It should line-up correctly when tab spaces are set to 4 (Github seems to be using 8); I aligned it to the opening parenthesis so it is immediately clear it is a part of the argument list and not some random nonsense.

Reply:
You could just move the first argument down a line to align everything with two tabs. Not everyone uses tabs at 4 spaces.
```c
	m_leftRightScroll->setGeometry(
		WHITE_KEY_WIDTH,
```

Reply:
Huh, I never though of this variant. It adds one more line, but the trailing `(` at the end of the line is easy to notice at a glance so it is clear what's going on.

I kind of assumed that https://github.com/LMMS/lmms/issues/4810 already happened and we agreed to use 4-spaces large tabs, but it seems it never made it to the conventions.

Your version is still better though, since it does not require any spaces for alignment, so thanks for the suggestion.